### ISSUE TITLE
Improve repo

### ISSUE THREAD

**Issue by AlexBerry0** [AlexBerry0/dots#10]
- [x] Move Todos to issues
- [x] Improve wording for README
- [x] Add screenshot of homarr for server screenshot
- [x] Change hosts description to a table
- [ ] Add propper commenting throughout nix files
- [x] Check README for spelling and grammer mistakes
- [ ] Make ISOs

**Comment by AlexBerry0** [comment]
Probably not going to do ISOs because the current method of distrobution is fine, and literally noone would use them. So now I just need to improve commenting and add the section in the README about my desktop, which needs to be finished first.

**Comment by AlexBerry0** [comment]
Honestly this is not going to get done completely for a long time, I understand my code and comment when I don't, and other people don't use these dots. I'll move this to a separate issue as I've done everything else here though.
